packages/dynamite/dynamite_end_to_end_test/pubspec.yaml

```diff
@@ -4,14 +4,18 @@ description: Tests for dynamite. Not meant for publishing.
 version: 1.0.0
 
 environment:
-  sdk: '>=3.1.1 <4.0.0'
+  sdk: '>=3.1.0 <4.0.0'
 
 dependencies:
-  built_value: ^8.6.2
+  built_collection: ^5.0.0
+  built_value: ^8.0.0
```

Comment:
Not related to this PR but we should probably pin the dependencies of non publishable packages to the latest version or use `any` with a up to date lockfile.

This way we could test the code dynamite generates against the latest version.
I noticed something similar in the `neon_lints` package as I don't think renovate would handle the lint_maker update (#1021).

Reply:
I thought about this as well. I think there is no way to build rules for renovate to do this automatically, but we could make a separate rule to enable non-major bumps for some packages specifically.

Reply:
But for your lint_maker changes it is fine since dev_dependencies are always bumped to the latest version.